### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Michael21107**
minor style edits

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ ExternalPoliceComputer.Functions.SendMessage("1st Line");
 ExternalPoliceComputer.Functions.SendMessage("2nd Line");
 ```
 
-Add the ExternalPoliceComputer.dll as a reference in your project. To prevent crashes, you have to check if EPC is available ([Example by opus49](https://github.com/Immersive-Plugins-Team/CalloutInterfaceAPI/blob/master/CalloutInterfaceAPI/Functions.cs#L26)).
+You can add the ExternalPoliceComputer.dll as a reference in your project. To prevent crashes, you have to check if EPC is available ([Example by opus49](https://github.com/Immersive-Plugins-Team/CalloutInterfaceAPI/blob/master/CalloutInterfaceAPI/Functions.cs#L26)).
```
Comment (jullevistrunz) [inline README.md]: No, you have to add it as a reference. Otherwise, VS will throw an error

File: README.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ You can edit all words used by EPC to your liking
 
 - File: `EPC/custom.css`
 
-For simple customization you may change colors, size, etc. in the `:root` selector. You can also add more css. `EPC/custom.css` will overwrite `EPC/main/styles.css`
+You may change colors, size, etc. in the `:root` selector for simple customization. You can also add more css. `EPC/custom.css` will overwrite `EPC/main/styles.css`
```
Comment (jullevistrunz) [inline README.md:72]: Be honest, does it really matter?

**Review by jullevistrunz — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Michael21107) [reply README.md:72]: not really

File: README.md (hunk 1 above)
Comment (Michael21107) [reply README.md]: fixed (i hope)

**Review by jullevistrunz — COMMENTED** [review]
There's one typo. PR with that one, only that one and not anything else that you think looks better.

**Comment by Michael21107** [comment]
what typo exactly, please?

File: README.md [hunk 3]
```diff
@@ -86,7 +86,7 @@ You can add your own JS code if you know what you're doing. Versions made by me
 
 ## API
 
-If you're callout dev, you can send messages to EPC's callout page once the callout has been accepted
+If you're a callout dev, you can send messages to EPC's callout page once the callout has been accepted
```
Comment (jullevistrunz) [inline README.md:89]: this one

Comment (Michael21107) [reply README.md:89]: gotcha

**Pull request closed without merging**